<ui version="4.0" >
 <class>QueueDlg</class>
 <widget class="QDialog" name="QueueDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Queue properties</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Name:</string>
     </property>
     <property name="buddy" >
      <cstring>lineName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineName" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Down speed limit (kiB/s):</string>
     </property>
     <property name="buddy" >
      <cstring>spinDown</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QSpinBox" name="spinDown" />
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Up speed limit (kiB/s):</string>
     </property>
     <property name="buddy" >
      <cstring>spinUp</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QSpinBox" name="spinUp" />
   </item>
   <item row="2" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3" >
    <widget class="QGroupBox" name="groupLimitQueue" >
     <property name="title" >
      <string>Limit active transfer count</string>
     </property>
     <property name="checkable" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Maximum active download transfers:</string>
        </property>
        <property name="buddy" >
         <cstring>spinTransfersDown</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QSpinBox" name="spinTransfersDown" />
      </item>
      <item row="0" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>Maximum active upload transfers:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QSpinBox" name="spinTransfersUp" />
      </item>
      <item row="1" column="2" >
       <widget class="QCheckBox" name="checkUpAsDown" >
        <property name="text" >
         <string>Count among downloads</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineName</tabstop>
  <tabstop>spinDown</tabstop>
  <tabstop>spinUp</tabstop>
  <tabstop>spinTransfersDown</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QueueDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>236</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QueueDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>304</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineName</sender>
   <signal>returnPressed()</signal>
   <receiver>QueueDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>474</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel" >
     <x>144</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkUpAsDown</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinTransfersUp</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>463</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel" >
     <x>276</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
